packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -536,6 +536,8 @@ export class ConnectionManager implements IConnectionManager {
 		let connection: IDocumentDeltaConnection | undefined;
 
 		if (docService.policies?.storageOnly === true) {
+			// go through all steps
+			this.props.establishConnectionHandler(reason);
```

Comment:
Could we have moved the original establishConnectionHandler call from line 559 to 537?

Reply:
That's a good point! :)